sql/npc_list.sql

```diff
@@ -21582,14 +21582,14 @@ INSERT INTO `npc_list` VALUES (17523346,'blank','     ',0,0.000,0.000,0.000,0,50
 INSERT INTO `npc_list` VALUES (17523347,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17523348,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17523349,'qm','???',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,NULL,1);
--- NC: INSERT INTO `npc_list` VALUES (17523350,'NOT_CAPTURED','Volto Oscuro',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17523350,'Volto_Oscuro','Volto Oscuro',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x010002038411E620E630E640E650406100700000,32,NULL,0);
 INSERT INTO `npc_list` VALUES (17523351,'csnpc','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x010002038411E620E630E640E650406100700000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17523352,'csnpc','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x010002038411E620E630E640E650406100700000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17523353,'csnpc','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17523354,'csnpc','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17523355,'csnpc','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17523356,'csnpc','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17523357,'qm','???',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00000C0200000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17523352,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17523353,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17523354,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17523355,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00008D0900000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17523356,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x00000C0200000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17523357,'qm','???',0,0.000,0.000,0.000,0,50,50,0,0,0,2,2051,0x0000340000000000000000000000000000000000,0,NULL,1);
```

Comment:
This qm has incorrect data, from retail:
`INSERT INTO `npc_list` VALUES (17523357,'???','???',148,-638.506,0.399,-422.900,32769,50,50,0,0,96,0,27,0x0000160900000000000000000000000000000000,34,NULL,1);`

Reply:
ok changing it

Reply:
Just to clarify as well, the first '???' can be qm for internal name, since we're not enumerating them yet.  I just pulled that from retail this morning